guides/doc-Release_notes/topics/katello-4.4.0.adoc

```diff
@@ -2,87 +2,119 @@
+ * Add repositories button should highlight in Content view (https://projects.theforeman.org/issues/34494[#34494], https://github.com/Katello/katello.git/commit/6ad4bc6feb55d45fba938abbdb6ceb50320010b4[6ad4bc6f])
+ * Create new content view should redirect to "Repositories" and not "Versions" tab (https://projects.theforeman.org/issues/34491[#34491], https://github.com/Katello/katello.git/commit/ff476372ba70dc3c8f991cac5bcccf5a35e0d7c2[ff476372])
+ * Extend CV API to support bulk removing versions (https://projects.theforeman.org/issues/34168[#34168], https://github.com/Katello/katello.git/commit/9e6b0d07ef198f7a0d55399e536844cb0908150b[9e6b0d07])
+
+=== Web UI
+ * [SAT-4229] Module streams - basic table (https://projects.theforeman.org/issues/34382[#34382], https://github.com/Katello/katello.git/commit/20e8293fb58fb2a9a9ef43b7fdd37a78103d558a[20e8293f])
+ * New Host details overview - Installable errata card (https://projects.theforeman.org/issues/33083[#33083])
+
 === Repositories
+ * Add remote download rate limit (https://projects.theforeman.org/issues/34345[#34345], https://github.com/Katello/katello.git/commit/7776b81ba39ded5eee3d7cc60dd793640b56132f[7776b81b])
  * Use the APT verbatim publisher for deb content on Pulp 3 foreman-proxy syncs (https://projects.theforeman.org/issues/34279[#34279], https://github.com/Katello/katello.git/commit/4a363fd945d56ad48f6a30446d9c16320819d4a5[4a363fd9])
+ * Add default download policy for deb content (https://projects.theforeman.org/issues/34119[#34119], https://github.com/Katello/katello.git/commit/48c128db02a1d9e95a81e371f4d9229ae25ebb5b[48c128db])
 
 === Hosts
+ * [SAT-4229] Content - Packages - Bulk select & remove (https://projects.theforeman.org/issues/34220[#34220], https://github.com/Katello/katello.git/commit/5e96a6f83e47ce3fe56862bde67d0d0b1c92bcb7[5e96a6f8])
  * Hosts - Change content source (https://projects.theforeman.org/issues/34211[#34211], https://github.com/Katello/katello.git/commit/c5f8e9532e61f9e5c7ec648decf81660628dd4ba[c5f8e953])
+ * [SAT-4229] Packages - Install packages modal (https://projects.theforeman.org/issues/34191[#34191], https://github.com/Katello/katello.git/commit/d433a901c0247b699ff742791367ecd73dd304e5[d433a901])
  * Katello host detail tabs should accept URL params for search (https://projects.theforeman.org/issues/34157[#34157], https://github.com/Katello/katello.git/commit/a12116ef4cb843c87ca9f149b1b573cd14afb03c[a12116ef])
- * Packages - Filter by status (https://projects.theforeman.org/issues/34131[#34131], https://github.com/Katello/katello.git/commit/38e14d1e02d73b430ca97fd813c6ac397ac707ee[38e14d1e])
+ * [SAT-4229] Packages - Filter by status (https://projects.theforeman.org/issues/34131[#34131], https://github.com/Katello/katello.git/commit/38e14d1e02d73b430ca97fd813c6ac397ac707ee[38e14d1e])
```

Comment:
I'd strongly urge Katello developers to leave these out of of titles. IMHO it's a bad pattern. There are a few occurrences so either humans are doing this or a script. Either way, I'd find that out.

In this particular case I modified the issue (https://projects.theforeman.org/issues/34131#note-8). Would you mind doing the same for others and regenerating the changelog?